src/components/user/SignUpForm/styles.js

```diff
@@ -25,6 +25,7 @@ const styles = StyleSheet.create({
   buttonText: {
     color: whiteColor,
     fontSize: 11,
+    fontFamily: 'OpenSans-Semibold',
```

Comment:
you should have default fonts defined as constants that don't depend on the actual font like `primaryFont`, `secondaryFont`, ... so if you need to change it later on it's very easy